test/bindings.test.ts

```diff
@@ -406,10 +406,15 @@ describe('MongoCryptConstructor', () => {
 
 describe('MongoCryptContext', () => {
   let context: MongoCryptContext;
+  let weakMongoCryptRef: WeakRef<MongoCrypt>;
+
   beforeEach(() => {
-    context = new MongoCrypt({
+    let crypt = new MongoCrypt({
       kmsProviders: serialize({ aws: {} })
-    }).makeDecryptionContext(serialize({}));
+    });
```

Comment:
Might be worthwhile adding a similar test for `MongoCryptKMSRequest.addResponse()` while we're changing how MongoCryptKMSRequest manages its internal state.  (I can also add one and push to this PR if you'd like).

Reply:
Hm – how would we test that without a working (i.e. hooked up to an actual KMS and MongoDB database) libmongocrypt instance?

Reply:
I hadn't thought much about it, but after trying a bit today I don't think it's straightforward.